ci-operator/config/openshift/cloud-provider-azure/openshift-cloud-provider-azure-main.yaml

```diff
@@ -115,11 +123,31 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure
+- as: e2e-azurestack-ipi
+  cluster: build01
```

Comment:
Do we want to stick it to build01? or only build01 has access to ash infra?

Reply:
I am not sure, I noticed both installer team and storage team are using build01

Reply:
okay, checked with installer team, only build01 is able to setup vpn which is required for ash and nutanix.